eyeshade/controllers/accounts.js

```diff
@@ -242,9 +243,65 @@ v1.getPaidTotals =
   }
 }
 
+/*
+  PUT /v1/accounts/{payment_id}/transactions/ads/{token_id}
+*/
+v1.adTransactions = {
+  handler: (runtime) => async (request, reply) => {
+    const {
+      params,
+      payload
+    } = request
+    const {
+      postgres
+    } = runtime
+    const {
+      amount: payloadAmount
+    } = payload
+    if (!_.isString(payloadAmount)) {
+      return reply(boom.badRequest())
+    }
+    const amount = (new BigNumber(payloadAmount)).dividedBy(1e18).toString()
```

Comment:
let's take amount not in probi - this matches the other accounts APIs

Reply:
fair enough.